### ISSUE TITLE
rai_node 9.0 fails to run on Centos 7 due to GLIB_2.18 dependency.

### ISSUE THREAD

**Issue by Seep1959** [nanocurrency/raiblocks#392]
I downloaded the rai_node 9.0 from the releases page. Upon running it I get the following output

```
# rai_node --diagnostics
rai_node: /lib64/libc.so.6: version `GLIBC_2.18' not found (required by rai_node)
```
uname output

```
Linux hostname.localhost 3.10.0-693.11.1.el7.x86_64 #1 SMP Mon Dec 4 23:52:40 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

/etc/os-release
```
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```
For reference this is a $5 digital oceans centos 7 VM. I had tried building from source but compiling boost was so taxing on its CPU it had to be hard reset.

**Comment by androm3da** [comment]
We should be able to make a binary build against centos 7.

**Comment by AndrewSChapman** [comment]
Bump - that would be awesome - I'd like to run a node on my CentOs machine.

**Comment by JohnWinner** [comment]
BUMP!
Same problem with CentOS 6.9
I don't feel like messing around with dependencies and compiling stuff. I would appreciate a binary.
Thanks!

**Comment by rkeene** [comment]
This will be resolved as part of #12 

Duplicate of #1091
